### PR TITLE
Setup echidna

### PR DESCRIPTION
**Pull request by aparnakr**
High Level:
This PR sets up the environment to write echidna tests. 

Specific Changes:
Created a unit tests folder
Created an echidna folder

### CODE REVIEW

File: contracts/echidna/mocks/EchidnaMockAddressBook.sol [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+contract EchidnaMockAddressBook {
+    constructor() public {}
+
+    function getController() external view returns (address) {
+        return 0x00a329C0648769a73afAC7F9381e08fb43DBEA70;
```
Comment (antoncoding) [inline contracts/echidna/mocks/EchidnaMockAddressBook.sol:8]: Will hardcoding this cause any problem?

Comment (aparnakr) [reply contracts/echidna/mocks/EchidnaMockAddressBook.sol:8]: it is hardcoded to return the owner address for now, which it thinks is the controller. This is to test the "onlyController" functionality.

**Review by antoncoding — APPROVED** [review]

File: contracts/echidna/mocks/EchidnaMockAddressBook.sol (hunk 1 above)
Comment (antoncoding) [reply contracts/echidna/mocks/EchidnaMockAddressBook.sol:8]: Got it I see it now. I misunderstood it as a hardcoded accounts value that might be different if we run truffle with different seed phrase 😹

**Review by haythem96 — COMMENTED** [review]
Can we remove `PropertiesOtokenTransferable` ? 
Also why `/tests/unit-test/` ?

**Review by antoncoding — APPROVED** [review]

**Comment by antoncoding** [comment]
Reopen a new PR without touching unit tests

**Pull request closed without merging**